tests/acceptance/features/bootstrap/WebUIUsersContext.php

```diff
@@ -153,19 +153,19 @@ public function theAdminCreatesAUserUsingWithoutAPasswordTheWebUI(
 	
 	/**
 	 *
-	 * @When the administrator deletes the group named :name using the webUI
+	 * @When the administrator deletes the group named :name using the webUI and confirms clicking yes
 	 *
 	 * @param string $name
 	 *
 	 * @return void
 	 */
 	public function theAdminDeletesTheGroupUsingTheWebUI($name) {
-		$this->usersPage->deleteGroup($name, $this->getSession());
+		$this->usersPage->deleteGroup($name, $this->getSession(), true);
 		$this->featureContext->rememberThatGroupIsNotExpectedToExist($name);
 	}
 
 	/**
-	 * @When the administrator deletes these groups using the webUI:
+	 * @When the administrator deletes these groups and confrims clicking yes using the webUI:
```

Comment:
"and confirms the deletion using the webUI"

(note the typo in "confrims" needs to be fixed while editing)